includes/apple-exporter/components/class-pre.php

```diff
@@ -0,0 +1,82 @@
+<?php
+namespace Apple_Exporter\Components;
+
+/**
+ * An HTML's blockquote representation.
+ *
+ * @since 0.2.0
+ */
+class Pre extends Component {
+
+  /**
+   * Look for node matches for this component.
+   *
+   * @param DomNode $node
+   * @return mixed
+   * @static
+   * @access public
+   */
+  public static function node_matches( $node ) {
+    if ( 'pre' == $node->nodeName ) {
```

Comment:
I'm actually not sure here. 

My worry is that matching `<code>` elements here will then break them into their own style component thing like we're currently doing with `<pre>`, which is not desireable.

We wouldn't want...

```
<p>This is a sentence with <code>some code</code> in it.</p>
```

to turn into...

```
styled json component thing
  This is a sentence with 
styled json component thing
  some code
styled json component thing
  in it.
```

I think just stripping the code tags is better than breaking a sentence.